testassets/InteropTestsNativeWebsite/InteropServer.cs

```diff
@@ -0,0 +1,81 @@
+﻿#region Copyright notice and license
```

Comment:
Add a readme.md to the InteropTestsNativeServer directory that will describe that this is a copy of InteropServer logic from grpc/grpc to allow for easier testing in the initial phase of managed client development and that we plan to remove it the future (how long are you planning to keep it)?

Also add a note that the authoritative C# interop server and the server here might diverge (e.g. if new functionality is added) in which case the Grpc.Core interop server code is the source of truth.

Once we have this readme, I think I'm fine with adding the native interop server.